base/show.jl

```diff
@@ -469,6 +469,11 @@ end
 isidentifier(s::Symbol) = isidentifier(string(s))
 
 isoperator(s::Symbol) = ccall(:jl_is_operator, Cint, (Cstring,), s) != 0
+"""
+    operator_precedence(s::Symbol)
+
+Returns an integer representing the precdedence of a Julia operator, relative to other operators. Higher numbers take precedence over lower-numbered operators. Returns `0` if a symbol is not a valid operator.
```

Comment:
Oops, sorry. That's what I get for trying to use Github's built-in editor. I've fixed it up now. Thanks.